package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "html-reporter",
   "version": "9.17.2",
-  "description": "Plugin for hermione which is intended to aggregate the results of tests running into html report",
+  "description": "Plugin for Testplane (ex-hermione) which is intended to aggregate the results of tests running into html report",
   "files": [
     "build"
   ],
   "exports": {
+    ".": "./build/index.js",
```

Comment:
Here as well I am strongly against making testplane a default export. I think it should be testplane, for future extensibility to another tools. If you don't agree, we can discuss this with the team.